From e6449f886fd476d06daca4de2fb2f657873b8d92 Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <masahiroy@kernel.org>
Date: Sat, 22 Jul 2023 13:47:53 +0900
Subject: [PATCH 11/62] kbuild: rpm-pkg: replace $__KERNELRELEASE in spec file
 with %{version}

${version} will be replaced with the value of the Version field.

Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
---
 scripts/package/mkspec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/package/mkspec b/scripts/package/mkspec
index 0befb4e2a..a6a383aaa 100755
--- a/scripts/package/mkspec
+++ b/scripts/package/mkspec
@@ -81,12 +81,12 @@ $S	BuildRequires: gcc make openssl openssl-devel perl python3 rsync
 	glibc package.
 
 $S$M	%package devel
-$S$M	Summary: Development package for building kernel modules to match the $__KERNELRELEASE kernel
+$S$M	Summary: Development package for building kernel modules to match the %{version} kernel
 $S$M	Group: System Environment/Kernel
 $S$M	AutoReqProv: no
 $S$M	%description -n kernel-devel
 $S$M	This package provides kernel headers and makefiles sufficient to build modules
-$S$M	against the $__KERNELRELEASE kernel package.
+$S$M	against the %{version} kernel package.
 $S$M
 $S	%prep
 $S	%setup -q -n linux
-- 
2.42.0

